app/assets/javascripts/cart.js.coffee

```diff
@@ -45,6 +42,8 @@ $(document).ready ->
       $('#modal_place').modal('show')
       $('#modal_place .yes-button').data('srid', srid)
       $('#modal_place .yes-button').data('service-id', id)
+      $('#modal_place .yes-button').on 'click', (e) ->
```

Comment:
Pretty sure this onClick handler can only run once. Whole modal + .yes-button re-rendered on line 41.